drivers/SmartThings/zigbee-button/src/thirdreality/init.lua

```diff
@@ -0,0 +1,58 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+
+local function present_value_attr_handler(driver, device, value, zb_rx)
+  local event
+  local additional_fields = {
+    state_change = true
+  }
+  if value.value == 0x0001 then
```

Comment:
nit: for readability you can create constants for these values at the top of the file and use them here

For example:
```
local HELD = 0x0000
local PUSHED = 0x0001
local DOUBLE = 0x0002
```